packages/shared/src/index.ts

```diff
@@ -6,3 +6,4 @@ export * from './core/index.js'
 export * from './workspaces/index.js'
 export * from './onboarding/index.js'
 export * from './limit/utils.js'
+export * from './blobs/index.js'
```

Comment:
we should stop exporting everything through the main barrel file and create export maps for each area (e.g. blobs) of @speckle/shared - `@speckle/shared/blobs` -> `shared/src/blobs/index.ts`